app/javascript/components/patient/close_contacts/CloseContactModal.js

```diff
@@ -0,0 +1,253 @@
+              <Form.Control.Feedback className="d-block" type="invalid">
+                {this.state.errors['last_name']}
+              </Form.Control.Feedback>
+            </Form.Group>
+          </Row>
+          <Row>
+            <Form.Group as={Col} lg="12" controlId="primary_telephone">
+              <Form.Label className="input-label">Phone Number {schema?.fields?.primary_telephone?._exclusive?.required && '*'}</Form.Label>
+              <PhoneInput
+                id="primary_telephone"
+                className="form-square"
+                value={this.state.primary_telephone}
+                onChange={this.handleChange}
+                isInvalid={!!this.state.errors['primary_telephone']}
+              />
+              <Form.Control.Feedback className="d-block" type="invalid">
+                {this.state.errors['primary_telephone']}
+              </Form.Control.Feedback>
+            </Form.Group>
+            <Form.Group as={Col} lg="12" controlId="email">
+              <Form.Label className="input-label">Email {schema?.fields?.email?._exclusive?.required && '*'} </Form.Label>
+              <Form.Control size="lg" className="form-square" value={this.state.email || ''} onChange={this.handleChange} />
+              <Form.Control.Feedback className="d-block" type="invalid">
+                {this.state.errors['email']}
+              </Form.Control.Feedback>
+            </Form.Group>
+          </Row>
+          <hr></hr>
+          <Row>
+            <Form.Group as={Col} lg="12" controlId="cc_last_date_of_exposure">
+              <Form.Label className="input-label">Last Date of Exposure {schema?.fields?.last_date_of_exposure?._exclusive?.required && '*'}</Form.Label>
+              <DateInput
+                id="cc_last_date_of_exposure"
+                date={this.state.last_date_of_exposure}
+                minDate={'2020-01-01'}
+                maxDate={moment().add(30, 'days').format('YYYY-MM-DD')}
+                onChange={date => this.handleDateChange(date)}
+                placement="top"
+                isInvalid={!!this.state.errors['last_date_of_exposure']}
+                customClass="form-control-lg"
+              />
+              <Form.Control.Feedback className="d-block" type="invalid">
+                {this.state.errors['last_date_of_exposure']}
+              </Form.Control.Feedback>
+            </Form.Group>
+            <Form.Group as={Col} lg="12" controlId="cc_assigned_user">
+              <Form.Label className="input-label">
+                Assigned User {schema?.fields?.assigned_user?._exclusive?.required && '*'}
+                <InfoTooltip tooltipTextKey="assignedUser" location="top"></InfoTooltip>
+              </Form.Label>
+              <Form.Control
+                isInvalid={this.state.errors['assigned_user']}
+                as="input"
+                list="assigned_users"
```

Comment:
is this necessary?

Reply:
yes

Reply:
I'm confused because it doesn't match the datalist id associated with it.  can we at least make it `assigned_users` to match?

Reply:
+1 the value for `list` and datalist `id` should match